docs/src/modules/components/MarkdownElement.js

```diff
@@ -23,6 +23,7 @@ const Root = styled('div')(({ theme }) => ({
     overflow: 'auto',
     WebkitOverflowScrolling: 'touch',
     maxWidth: 'calc(100vw - 32px)',
+    maxHeight: '350px',
```

Comment:
I wonder if he wouldn't want to apply it this style based on cases, for instance in https://deploy-preview-33100--material-ui.netlify.app/material-ui/migration/troubleshooting/#storybook-and-emotion, the scrollbar feels like it makes it harder to consume the content. I have noticed this first in https://deploy-preview-33595--material-ui.netlify.app/blog/aggregation-functions/#customize-everything

Reply:
I guess we can do it, but we will need to add some parameters when creating the code templates. cc @siriwatknp for opinion.